libmusic/src/lmtypes.cpp

```diff
@@ -62,6 +67,35 @@ std::ostream& operator<<(std::ostream& os, const chord_quality_t& q)
 {
     std::map<chord_quality_t, std::string> q2sMap;
 
+#if defined(CFG_HARTE_SYNTAX) && CFG_HARTE_SYNTAX == 1
+    q2sMap[cq_maj]          = "maj";
+    q2sMap[cq_min]          = "min";
+    q2sMap[cq_5]            = "5";
+    q2sMap[cq_7]            = "7";
+    q2sMap[cq_maj7]         = "maj7";
+    q2sMap[cq_min7]         = "min7";
+    q2sMap[cq_sus2]         = "sus2";
+    q2sMap[cq_sus4]         = "sus4";
+    q2sMap[cq_hdim7]        = "hdim7";
+    q2sMap[cq_aug]          = "aug";
+    q2sMap[cq_dim]         = "dim";
+    q2sMap[cq_dim7]         = "dim7";
+    q2sMap[cq_maj_add9]     = "maj(9)";
+    q2sMap[cq_min_add9]     = "min(9)";
+    q2sMap[cq_maj6]         = "maj6";
+    q2sMap[cq_min6]         = "min7";
+    q2sMap[cq_maj9]         = "maj9";
+    q2sMap[cq_min9]         = "min9";
+    q2sMap[cq_maj_add11]    = "maj(11)";
+    q2sMap[cq_7_add9sharp]  = "7(#9)";
+    q2sMap[cq_9]            = "9";
+    q2sMap[cq_aug7]         = "aug7";
+    q2sMap[cq_maj11]        = "maj11";
+    q2sMap[cq_min11]        = "min11";
+    q2sMap[cq_maj13]        = "maj13";
+    q2sMap[cq_min13]        = "min13";
+#else
+#error "Non-Harte chord types string representations need to be updated"
```

Comment:
looks like non-Harte configuration is broken. If we are to support both representation we shouldn't leave it like this.

Reply:
I don't consider it a priority at this point, that's why I deliberately placed a compilation error there. Will fix since you're insisting.

Reply:
This syntax does not produce unique template names, e.g.:
```
D:min,0,0,2.3094,0,0,1.1547,0,0,0,1.1547,0,0,0,0,2.3094,0,0,2.3094,0,0,0,2.3094,0,0
D:min,0,0,1.26491,0,0,1.26491,0,0,0,1.26491,0,0,0,0,2.52982,0,0,2.52982,0,0,0,2.52982,0,0
```

which makes it much harder to distinguish them in automated processing (e.g. SNR)

Reply:
It's not the syntax, it's the extra logic added to overload `<<` `Chord` operator. We should not print bass note if it matches a root note and this behavior is not correct for the `Chord` class.
If we need template labels to be unique, we should fix `operator<<` for `ChordTpl` instead.

Reply:
Addressed with the extra commit.